errolyn/a10.py

```diff
@@ -0,0 +1,35 @@
+
+bit_string = input("Please enter a 32-bit integer: ") # example: 10000000111111110000001100001111
+# there is no input sanitization or error handling since the assignment did not ask for it,
+# I would turn this into it's own function if that was a requirement.
+bit_array = [] # holds the raw bit sets in an array
+bit_parce =[] # holds parced values of bit_array
+
+def split_string(string):
+    while string: # this terminates when there is nothing left in "string".
```

Comment:
It might be nice to avoid modifying `string` inside the loop, as it's more state to follow. A for loop with an 8-count step can help:

```python
parts = []
for start in range(0, len(string), 8):
    parts.append(string[start:start+8])
```